pyserini/prebuilt_index_info.py

```diff
@@ -900,6 +900,19 @@
         "unique_terms": 8722502,
         "downloaded": False
     },
+    "wiki-all-6-3-tamber": {
+        "description": "Lucene index of wiki-all-6-3-tamber from castorini/odqa-wiki-corpora",
+        "filename": "index-wiki-all-6-3-tamber-20230111-40277a.tar.gz",
```

Comment:
Can we create a README associated with both entries and link to them?

That way, we can make the description more compact.

Reply:
done